troposphere/static/js/components/admin/ResourceRequest/ConfirmApproveModal.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+import BootstrapModalMixin from "components/mixins/BootstrapModalMixin";
+
+
+const ConfirmApproveModal = React.createClass({
+    propTypes: {
+        onApprove: React.PropTypes.func.isRequired
+    },
+    onCancel() {
+        this.hide();
+        this.props.onCancel();
```

Comment:
I realize that `onCancel` may not be required, but it is missing from `propTypes`. 

Not required for the merge, but just something of note for me (it's unclear: require, or !required).

Reply:
I'll add it.

Reply:
Fixed